src/Forms/HTMLEditor/HTMLEditorField.php

```diff
@@ -185,4 +190,12 @@ public function Field($properties = array())
         $this->getEditorConfig()->init();
         return parent::Field($properties);
     }
+
+    public function getSchemaStateDefaults()
+    {
+        $stateDefaults = parent::getSchemaStateDefaults();
+        $config = $this->getEditorConfig();
+        $stateDefaults['data'] = $config->getConfigSchemaData();
```

Comment:
May as well just write `$stateDefaults['data'] = $this->getEditorConfig()->getConfigSchemaData()`